src/js/controller/controller.js

```diff
@@ -274,10 +274,7 @@ define([
 
             function _play(meta) {
                 var status;
-
-                if (meta) {
-                    _model.set('playReason', meta.reason);
-                }
+                var playReason = meta ? meta.reason : 'unknown';
```

Comment:
Let's leave playReason in the other events. Model playReason could be set to unknown here.